pkg/k8s/apis/cilium.io/v2/validator/validator.go

```diff
@@ -157,19 +137,13 @@ func checkWildCardToFromEndpoint(ccnp *unstructured.Unstructured) error {
 	// Print the warninig only once per CCNP.
 	if resCCNP.Spec != nil {
```

Comment:
I think that we may be able to remove the entire function `checkWildCardToFromEndpoint()` now actually, not just the log.

Reply:
absolutely